CWE/localisation/Chrisfox's events.csv

```diff
@@ -54,63 +54,63 @@ EVTDESC8012111;The DPRK has an interesting ideology - it incorporates elements o
+EVTDESC8023012104;With Gheorghiu-Dej's grip on Romania having weakened with the death of Stalin and pressure from a now anti-Stalinist Moscow, forces within the political apparatuses of the nation and party have moved quickly to replace the old defender of the Stalinist order. Pauker, already having been in a position of power and prominence, has rallied her support within the 'Moscow Faction' and courted enough moderate support to ensure Gheorghiu-Dej's ouster without issue - as well as to secure her own place within Romanian politics.;;;;;;;;;;;;;x;;;;;;;;;;;;;;;;
+EVTDESC8023012105;Gheorghiu-Dej's death means the passing of the torch of Romanian politics to another within the party.  Apostol's remaining support within the party and prominence within Romania mean a more competitive succession. Already both of the prospective candidates are attempting to rally enough support to be nominated as the next First Secretary. While Ceaucescu seems to have a slight advantage due to infighting, Apostol remains a solid choice for many who hope to continue the competent stability of the previous decade with a more tested hand.;;;;;;;;;;;;;x;;;;;;;;;;;;;;;;
+EVTDESC8023012106;Even before her rise to now, Ana Pauker had helped facilitate the emigration of several thousand Romanian Jews to the new state of Israel - defying the Stalinist norm at the time that saw increasing restrictions on both emigration and immigration. While many Jews remained in Romania, her efforts had earned her ire from political rivals and praise from her political allies and the Jewish community itself.\n\nHowever, now that she is no longer challenged by those same Stalinist forces the opportunity has arisen to provide a 'Right of Return' to all of Romania's remaining Jews to their new homeland.;;;;;;;;;;;;;x;;;;;;;;;;;;;;;;
+EVTDESC8023012107;With power in Romania now passing to anti-Stalinists, the time has come to purge the late Soviet leader's influence from within the party and Romanian society. Ana Pauker - initially seen as supportive of the direction of the new government before turning out of its favor with increasingly vocal opposition - has today announced a campaign of liberalization within the party, with even more far-reaching plans for strengthening the 'Moscow Faction' influence.;;;;;;;;;;;;;x;;;;;;;;;;;;;;;;
+EVTDESC8023012108;The 'Magyar Wing' of Romania's Communist Party was dissolved in 1953, just a year after the creation of the 'Magyar Autonomous Region' in Romania. It had provided one of the few mechanisms for Hungarians in Romania to fight against the increasing policy of 'Romanianization' from the Nationalists and Stalinists that had helped prop up the rule of Gheorghiu-Dej. \n\nGiven that the Stalinists no longer wield much political influence - the question has been brought up by more liberal-minded party members as to a potential revival of this institution.;;;;;;;;;;;;;x;;;;;;;;;;;;;;;;
```

Comment:
I think the localisation file has been encoded wrongly. It should be set as ANSI.

Reply:
That's strange, opening the file in Notepad++ says that its already in ANSI, even when I download a 'fresh' version of the file.